### PR TITLE
added tutorial on uploading

### PR DESCRIPTION
**Pull request by sadjava**

### CODE REVIEW

File: docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Uploading data and model checkpoints to s3
+There are two ways to upload them: 
+1. using python script
```
Comment (QazyBi) [inline docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md:3]: by using python script you use the command line interface. I believe you wanted to write Application Programming Interface(API)

File: docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+2. from cli (command line interface)
+
+Both optinions will be considered.
+
+For both uploading data and model using either cli or python script you can omit access_key and secret_key arguments, but they will be asked. So you need to specify them anyway.
```
Comment (QazyBi) [inline docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md:8]: "When uploading data or a model using either the command line interface (CLI) or an API, you have the option to omit the access_key and secret_key arguments. The framework will search for these values in the `~/.aws/credentials` and `.env` files. However, if these values are not found, you will be prompted in CLI."

File: docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            ckpt_path = "pretrained/best.pkl",
+            remote_save_path = "https://api.blackhole.ai.innopolis.university/pretrained/model.pickle",
+            metrics = {"some metric": 0.04},
+            access_key = "access key",
+            secret_key = "secret key")
+```
+### CLI
+To upload your checkpoint you can also run this command.
+``` bash
+python innofw/zoo/uploader.py --ckpt_path pretrained/best.pkl
```
Comment (QazyBi) [inline docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md:86]: In shell, to achieve line continuation, you typically use a backslash(\) at the end of each line.

File: docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                              --experiment_config_path classification/config.yaml
+                              --remote_save_path https://api.blackhole.ai.innopolis.university/pretrained/testing/lin_reg_house_prices.pickle
+                              --access_key access_key --secret_key secret_key
+```
```
Comment (QazyBi) [inline docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md:90]: You need to add here metrics
```
--metrics '{"some metric": 0.04}'
```

File: docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+``` bash
+python innofw/data_mart/uploader.py --dataset_config_path classification/config.yaml
+                                    --remote_save_path  https://api.blackhole.ai.innopolis.university/public-datasets/test_dataset/ 
+                                    --access_key access_key --secret_key secret_key\
+```
+## Uploading model checkpoint to s3
+To upload your checkpoint you need to specify config file of your experiment that must be stored in <b>config/experiments</b> folder, path to checkpoint and metrics you want to save.
+### Python
+You can run this python script to upload checkpoint with appropriate values.
```
Comment (QazyBi) [inline docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md:71]: You can call this Python function to upload a checkpoint with appropriate values.

File: docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+### CLI
+To upload your data to s3 you can also run this command in cli.
+
+``` bash
+python innofw/data_mart/uploader.py --dataset_config_path classification/config.yaml
+                                    --remote_save_path  https://api.blackhole.ai.innopolis.university/public-datasets/test_dataset/ 
+                                    --access_key access_key --secret_key secret_key\
+```
+## Uploading model checkpoint to s3
+To upload your checkpoint you need to specify config file of your experiment that must be stored in <b>config/experiments</b> folder, path to checkpoint and metrics you want to save.
```
Comment (QazyBi) [inline docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md:69]: To upload your checkpoint, you need to specify the config file for your experiment, which should be stored in the <b>config/experiments</b> folder. Additionally, provide the path to the checkpoint and specify the metrics you want to save.

File: docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+                remote_save_path="https://api.blackhole.ai.innopolis.university/public-datasets/test_dataset/",
+                access_key = "access key",
+                secret_key = "secret key")
+```
+
+### CLI
+To upload your data to s3 you can also run this command in cli.
+
+``` bash
+python innofw/data_mart/uploader.py --dataset_config_path classification/config.yaml
```
Comment (QazyBi) [inline docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md:64]: In shell, to achieve line continuation, you typically use a backslash() at the end of each line.

**Review by QazyBi — CHANGES_REQUESTED** [review]

**Comment by sadjava** [comment]
I guess I'm done. Check again please.

**Pull request merged**